test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/CanvasViewSamsple.cs

```diff
@@ -0,0 +1,162 @@
+using Tizen.NUI.BaseComponents;
+using Tizen.NUI.BaseComponents.VectorGraphics;
+using System.Collections.Generic;
+
+namespace Tizen.NUI.Samples
+{
+    using log = Tizen.Log;
+
+    public class CanvasViewSample : IExample
+    {
+        const string tag = "NUITEST";
+
+        private View root;
+        private CanvasView canvasView ;
+        private Shape shape;
+        private Timer timer;
+        private int count = 0;
+
+        public void Activate()
+        {
+            Window window = NUIApplication.GetDefaultWindow();
+
+            root = new View()
+            {
+                Size = window.Size,
+                ParentOrigin = ParentOrigin.Center,
+                PivotPoint = PivotPoint.Center,
+                PositionUsesPivotPoint = true,
+            };
+            window.Add(root);
+
+            canvasView = new CanvasView(window.Size)
+            {
+                Size = window.Size,
+                ParentOrigin = ParentOrigin.Center,
+                PivotPoint = PivotPoint.Center,
+                PositionUsesPivotPoint = true,
+            };
+
+            Shape shape1 = new Shape();
```

Comment:
After fix some methods to properties, is it possible to fix the samples like below,

```C#
Shape shape1 = new Shape()
{
  FillColor = new Color(0.5f, 1.0f, 0.0f, 1.0f),
  StrokeColor = new Color(0.5f, 0.0f, 0.0f, 0.5f),
  StrokeWidth = 10.0f
};

shape1.AddRect(-50.0f, -50.0f, 100.0f, 100.0f, 10.0f, 10.0f);
shape1.Translate(100.0f, 100.0f);
shape1.Scale(1.2f);
shape1.Rotate(45.0f);

canvasView.AddDrawable(shape1);
```

Reply:
Update sample code to use properties